packages/storybook/src/stories/grid-sorting.stories.tsx

```diff
@@ -256,8 +256,8 @@ export const CustomComparator = () => {
         params.getValue(params.id, 'age') || 'x'
       }`,
     sortComparator: (v1, v2, cellParams1, cellParams2) =>
-      cellParams1.api.getCellValue(cellParams1.id, 'age') -
-      cellParams2.api.getCellValue(cellParams2.id, 'age'),
+      (cellParams1.api.getCellValue(cellParams1.id, 'age') as number) -
```

Comment:
Why has this one change?

Reply:
Before, `api` was of type `any` so `cellParams1.api.getCellValue(cellParams1.id, 'age')` and `cellParams1.api.getCellValue(cellParams2.id, 'age')` was of type `any`

Now they are of type `GridCellValue`
So TypeScript complains that I am trying to do a substraction on a type that potentially does not support it (`object` for instance`)

I would like to add a generic to `getCellValue` (https://github.com/mui-org/material-ui-x/issues/2190) to do `cellParams1.api.getCellValue<number>(cellParams1.id, 'age')` instead of casting but it requires changes in the doc gen.